01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,11 @@
+[*1..20].each do |x|
```

Comment:
今だと「`1..20`というRangeクラスのオブジェクトをSplat演算子(*)で展開したものを用いて配列を初期化し、その配列に対して`each`」という処理になっていて意外とややこしい感じになってますね。
動作的には問題ないのですが、Rangeオブジェクトも`each`が使えるので下記のように書いた方が簡単です。

```suggestion
(1..20).each do |x|
```

Rangeについてのマニュアルも参考にしてみてくださいー。
https://docs.ruby-lang.org/ja/latest/class/Range.html